### ISSUE TITLE
Crafty.load fails with duplicate image

### ISSUE THREAD

**Issue by ashes999** [craftyjs/Crafty#764]
Tested in Crafty 0.6.2 on Ubuntu/Chrome.

Simplest case:

`Crafty.load(['image.png', 'image.png'], function() { console.log("Done!"); });`

The load-complete function never fires. Crafty fails silently and doesn't ever complete loading. Please at least raise an error highlighting the problem, so that I can more quickly track down the issue (ideally, mention the duplicate-loaded asset).

Another option is just loading and moving on, ignoring the fact that an asset was loaded twice (provided it doesn't actually download twice).

**Comment by mucaho** [comment]
So you don't get an error inside the [error callback](http://craftyjs.com/api/Crafty-loader.html)?

``` javascript
Crafty.load(['image.png', 'image.png'],
    function() { //when loaded
        console.log("Done!");
    },
    function(e) { //progress
    },
    function(e) { //uh oh, error loading
        console.log("Error!", e);
    }
);
```

**Comment by ashes999** [comment]
@mucaho correct, the error call-back never fires.

**Comment by heniotierra** [comment]
Fixed this for the new `Crafty.load` code. If the image(or audio) has already been loaded, it won't be loaded again, and oncomplete will still be called at the end, but no error is thrown in this case.

EDIT: A new PR was sent, this new one enabling onerror function to be called in this situation.

**Comment by mucaho** [comment]
should be fixed via a5fd9872eceb97d5a86e146d6bf2ebc5191f50c0 now
